src/main/java/com/julienvey/trello/impl/http/AsyncTrelloHttpClient.java

```diff
@@ -50,13 +50,13 @@ public void onThrowable(Throwable t) {
     }
 
     @Override
-    public <T> T postForObject(String url, T object, final Class<T> objectClass, String... params) {
+    public <T> T postForObject(String url, Object object, final Class<T> objectClass, String... params) {
         Future<T> f;
         try {
             byte[] body = this.mapper.writeValueAsBytes(object);
-            f = asyncHttpClient.preparePost(UrlExpander.expandUrl(url, params)).setBody(body).execute(
-                    new AsyncCompletionHandler<T>() {
-
+            f = asyncHttpClient.preparePost(UrlExpander.expandUrl(url, params))
+                    .setHeader("Content-Type", "application/json")
+                    .setBody(body).execute(new AsyncCompletionHandler<T>() {
```

Comment:
why is this manual header setting needed here? was there a specific bug ?

Reply:
Yes, there was a bug without this header. For some reasons, Trello API does not understand request body (when `POST`ing) without `Content-Type` header.

Reply:
let's have this in a separate branch if this is an old bug. it will be easier to review, merge and potentially roll back if needed

Reply:
Without that header newly added `addMemberToCard` method not working.

Reply:
alright, sounds good